src/lib/pdfconverter.cc

```diff
@@ -546,13 +546,26 @@ void PdfConverterPrivate::findLinks(QWebFrame * frame, QVector<QPair<QWebElement
 		uexternal  = PageObject::webPageToObject[frame->page()]->settings.useExternalLinks;
 	}
 	if (!ulocal && !uexternal) return;
-	foreach (const QWebElement & elm, frame->findAllElements("a")) {
+	foreach (QWebElement elm, frame->findAllElements("a")) {
```

Comment:
We need to call ```set()``` functions to inject ```nbsp;``` so ```const``` needs to be removed. The non-const reference gives me the following compiler error:

```
g++ -c -pipe -O2 -Wall -W -D_REENTRANT -fPIC -DQT4_STATICPLUGIN_TEXTCODECS -DVERSION=0.12.3.2 -DFULL_VERSION=0.12.3.2 -DBUILDING_DLL -DQT_NODLL -DQT_NO_DEBUG -DQT_WEBKIT_LIB -DQT_SVG_LIB -DQT_XMLPATTERNS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I/home/rpedela/wkhtmltopdf/qt/mkspecs/linux-g++ -I/home/rpedela/wkhtmltopdf/src/lib -I../../../qt/include/QtCore -I../../../qt/include/QtNetwork -I../../../qt/include/QtGui -I../../../qt/include/QtXmlPatterns -I../../../qt/include/QtSvg -I../../../qt/include/QtWebKit -I../../../qt/include -I/home/rpedela/wkhtmltopdf/include -I. -I/home/rpedela/wkhtmltopdf/src/lib -I. -o pdfconverter.o /home/rpedela/wkhtmltopdf/src/lib/pdfconverter.cc
In file included from ../../../qt/include/QtCore/qglobal.h:1:0,
                 from ../../../qt/include/QtCore/../../../../../qt/src/corelib/global/qnamespace.h:45,
                 from ../../../qt/include/QtCore/qnamespace.h:1,
                 from ../../../qt/include/QtCore/../../../../../qt/src/corelib/kernel/qobjectdefs.h:45,
                 from ../../../qt/include/QtCore/qobjectdefs.h:1,
                 from ../../../qt/include/QtCore/../../../../../qt/src/corelib/kernel/qobject.h:47,
                 from ../../../qt/include/QtCore/qobject.h:1,
                 from ../../../qt/include/QtCore/QObject:1,
                 from /home/rpedela/wkhtmltopdf/src/lib/converter.hh:24,
                 from /home/rpedela/wkhtmltopdf/src/lib/converter_p.hh:24,
                 from /home/rpedela/wkhtmltopdf/src/lib/pdfconverter_p.hh:24,
                 from /home/rpedela/wkhtmltopdf/src/lib/pdfconverter.cc:22:
/home/rpedela/wkhtmltopdf/src/lib/pdfconverter.cc: In member function 'void wkhtmltopdf::PdfConverterPrivate::findLinks(QWebFrame*, QVector<QPair<QWebElement, QString> >&, QVector<QPair<QWebElement, QString> >&, QHash<QString, QWebElement>&)':
../../../qt/include/QtCore/../../../../../qt/src/corelib/global/qglobal.h:2500:34: error: invalid initialization of non-const reference of type 'QWebElement&' from an rvalue of type 'const QWebElement'
     for (variable = *_container_.i;; __extension__ ({--_container_.brk; break;}))
                                  ^
../../../qt/include/QtCore/../../../../../qt/src/corelib/global/qglobal.h:2564:21: note: in expansion of macro 'Q_FOREACH'
 #    define foreach Q_FOREACH
                     ^
/home/rpedela/wkhtmltopdf/src/lib/pdfconverter.cc:549:2: note: in expansion of macro 'foreach'
  foreach (QWebElement & elm, frame->findAllElements("a")) {
  ^
make[1]: *** [pdfconverter.o] Error 1
```

Do you have any suggestions?